dust_extinction/averages.py

```diff
@@ -1056,3 +1057,103 @@ def evaluate(self, in_x):
         f = interp1d(self.obsdata_x, self.obsdata_axav)
 
         return f(x)
+
+class B1992(BaseExtAveModel):
```

Comment:
Please move the code for this model to right after `RL85_MWGC` to keep it in date order.  A convention to make it straightforward to find the model code.